DoctrineBundle.php

```diff
@@ -78,7 +77,7 @@ public function boot()
                 assert($registry instanceof Registry);
 
                 foreach ($registry->getManagers() as $em) {
-                    assert($em instanceof EntityManager);
+                    assert($em instanceof EntityManagerInterface);
```

Comment:
I don't see any `use` statement for this one?

Reply:
Yep, my bad. Storm decided not to remind me.